### PR TITLE
UI Docs update

### PR DESCRIPTION
**Pull request by nidzix**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: client/app/docs/module.js [hunk 1]
```diff
@@ -34,6 +34,31 @@
                 scope.closeModal = function() {
                     scope.isModalOpen = false;
                 };
+
+                //Select boxes
+                scope.opts = ['Serbia', 'Czech Republic', 'Germany', 'Australia'];
+
+                //Typeahead
+                scope.taTerms = ['Serbia', 'Czech Republic', 'Germany', 'Australia', 'Canada', 'Russia', 'Italy', 'Egypt', 'China'];
```
Comment (mugurrus) [inline client/app/docs/module.js]: Why hardcode countries?

**Comment by mugurrus** [comment]
:+1:

File: client/app/docs/module.js (hunk 1 above)
Comment (nidzix) [inline client/app/docs/module.js]: I don't know what is the other option? It's just an example how to use autocomplete element...

**Pull request merged**